tests/test_conftest_logging.py

```diff
@@ -0,0 +1,219 @@
+import os
+from unittest.mock import Mock
+
+from conftest import pytest_sessionstart
+
+
+def _simulate_configure_logging_logic(dotenv_values_return=None, existing_log_level=None, single_test_env=None):
```

Comment:
Remove all of these tests.

The tests I was referring to are the ones failing from:
- `ee/vellum_cli/tests/test_push.py`
- `ee/vellum_cli/tests/test_ping.py`
- `ee/vellum_cli/tests/test_image_push.py`

Fix those failing tests, while making no additional changes to our conftest